src/handlers/digitalOcean.go

```diff
@@ -0,0 +1,92 @@
+package handlers
+
+import (
+	"context"
+	"os"
+
+	"github.com/digitalocean/godo"
+)
+
+type Do_client struct {
+	client *godo.Client
+}
+
+type Config struct {
+	Name   string `json:"name"`
+	Region string `json:"region"`
+	Size   string `json:"size"`
+	Image  string `json:"image"`
+}
+
+type TokenRequest struct {
+	token string
+}
+
+type CreateDropletRequest struct {
+	Name string
+}
+
+type DropletListResponse struct {
+	DropletList interface{} `json:"DropletList"`
+}
+
+func (d *Do_client) authClient(ctx context.Context, req TokenRequest) (Config, *Error) {
+	client, err := godo.NewFromToken(os.Getenv("DIGITAL_OCEAN_TOKEN"))
```

Comment:
Instead you can create a function NewDoClient(client godo.Client) and build this client outside, then just pass a godo.Client to this function, as a result we have a client that's able to send requests